CHANGELOG.md

```diff
@@ -20,6 +25,18 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [4.0.0](https://github.com/Nike-Inc/Elevate/releases/tag/4.0.0)
+
+Released on 2019-04-30. All issues associated with this milestone can be found using this
+[filter](https://github.com/Nike-Inc/Elevate/milestone/10?closed=1).
+
+#### Updated
+
+- The project to support Swift 5 only and no longer support Swift 4.2.
+- Updated by [Dmitri Fuerle](https://github.com/dfuerle) in Pull Request [#38](https://github.com/Nike-Inc/Elevate/pull/38).
+- The deployment targerts to iOS 10.0, macOS 10.12, tvOS 10.0, and watchOS 3.0.
+- Updated by [Dmitri Fuerle](https://github.com/dfuerle) in Pull Request [#38](https://github.com/Nike-Inc/Elevate/pull/38).
```

Comment:
Could you add a couple spaces before both of the `- Updated by` lines so they get nested?